gradle.properties

```diff
@@ -34,7 +34,7 @@ spring_no_http_plugin_version=0.0.3.RELEASE
 checkstyle_version=8.24
 openapi_plugin_version=4.2.2
 
-xm_commons_version=2.2.41
+xm_commons_version=2.3.16
```

Comment:
If scheduler is used, consider migration to xm_commons_version=2.3.17

Reply:
for now not used

Reply:
done